src/System.Private.ServiceModel/src/windows/project.json

```diff
@@ -1,49 +1,49 @@
-    "System.Collections": "4.0.11-rc2-23712",
-    "System.Collections.Concurrent": "4.0.11-rc2-23712",
-    "System.Collections.NonGeneric": "4.0.1-rc2-23712",
-    "System.Collections.Specialized": "4.0.1-rc2-23712",
-    "System.ComponentModel": "4.0.1-rc2-23712",
-    "System.ComponentModel.EventBasedAsync": "4.0.11-rc2-23712",
-    "System.Diagnostics.Contracts": "4.0.1-rc2-23712",
-    "System.Diagnostics.Debug": "4.0.11-rc2-23712",
-    "System.Diagnostics.Tools": "4.0.1-rc2-23712",
-    "System.Diagnostics.Tracing": "4.0.21-rc2-23712",
-    "System.Globalization": "4.0.11-rc2-23712",
-    "System.IO": "4.0.11-rc2-23712",
-    "System.IO.Compression": "4.1.0-rc2-23712",
-    "System.Linq": "4.0.1-rc2-23712",
-    "System.Linq.Expressions": "4.0.11-rc2-23712",
-    "System.Linq.Queryable": "4.0.1-rc2-23712",
+    "System.Collections": "4.0.10",
+    "System.Collections.Concurrent": "4.0.0",
+    "System.Collections.NonGeneric": "4.0.0",
+    "System.Collections.Specialized": "4.0.0",
+    "System.ComponentModel": "4.0.0",
+    "System.ComponentModel.EventBasedAsync": "4.0.10",
+    "System.Diagnostics.Contracts": "4.0.0",
+    "System.Diagnostics.Debug": "4.0.0",
+    "System.Diagnostics.Tools": "4.0.0",
+    "System.Diagnostics.Tracing": "4.0.20",
+    "System.Globalization": "4.0.0",
+    "System.IO": "4.0.10",
+    "System.IO.Compression": "4.0.0",
+    "System.Linq": "4.0.0",
+    "System.Linq.Expressions": "4.0.0",
+    "System.Linq.Queryable": "4.0.0",
     "System.Net.Http": "4.0.1-rc2-23712",
-    "System.Net.Primitives": "4.0.11-rc2-23712",
-    "System.Net.WebHeaderCollection": "4.0.1-rc2-23712",
+    "System.Net.Primitives": "4.0.10",
+    "System.Net.WebHeaderCollection": "4.0.0",
     "System.Net.WebSockets": "4.0.0-rc2-23712",
     "System.Net.WebSockets.Client": "4.0.0-rc2-23712",
-    "System.ObjectModel": "4.0.11-rc2-23712",
-    "System.Reflection": "4.0.11",
-    "System.Reflection.DispatchProxy": "4.0.1-rc2-23712",
-    "System.Reflection.Extensions": "4.0.1-rc2-23712",
-    "System.Reflection.Primitives": "4.0.1-rc2-23712",
-    "System.Reflection.TypeExtensions": "4.0.1-rc2-23712",
-    "System.Resources.ResourceManager": "4.0.1-rc2-23712",
-    "System.Runtime": "4.0.21-rc2-23712",
-    "System.Runtime.Extensions": "4.0.11-rc2-23712",
-    "System.Runtime.Handles": "4.0.1-rc2-23712",
-    "System.Runtime.InteropServices": "4.0.21-rc2-23712",
+    "System.ObjectModel": "4.0.0",
+    "System.Reflection": "4.0.0",
+    "System.Reflection.DispatchProxy": "4.0.0",
+    "System.Reflection.Extensions": "4.0.0",
+    "System.Reflection.Primitives": "4.0.0",
+    "System.Reflection.TypeExtensions": "4.0.0",
+    "System.Resources.ResourceManager": "4.0.0",
+    "System.Runtime": "4.0.20",
+    "System.Runtime.Extensions": "4.0.10",
+    "System.Runtime.Handles": "4.0.0",
+    "System.Runtime.InteropServices": "4.0.0",
     "System.Runtime.Serialization.Xml": "4.0.10",
-    "System.Security.Claims": "4.0.1-rc2-23712",
```

Comment:
I commented in the wrong place -- would you mind making S.R.S.Xml be 4.1.0 to fix #462